subsys/net/lib/openthread/platform/entropy.c

```diff
@@ -19,22 +19,19 @@ LOG_MODULE_REGISTER(net_otPlat_entropy, CONFIG_OPENTHREAD_L2_LOG_LEVEL);
 #error OpenThread requires an entropy source for a TRNG
 #endif
 
+static const struct device *dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_entropy));
```

Comment:
Should this be a build error, for sure?

Reply:
Module is not operational without this device, whether it's a failure that's handled later: I don't know. We need maintainers voice here.

Reply:
I think it's ok to require at build time to have an entropy device present in the system, AFAIK there's no fallback mechanism in OT for that. @edmont What do you think?

Reply:
I agree.